projects/storefrontapp-e2e-cypress/cypress/integration/vendor/cds/profile-tag.e2e-spec.ts

```diff
@@ -21,6 +21,57 @@ describe('Profile-tag component', () => {
 
     profileTagHelper.triggerLoaded();
   });
+  it('should send a CartChanged event on adding an item to cart', () => {
+    goToProductPage();
+    cy.get('cx-add-to-cart button.btn-primary').click();
+    cy.get('cx-added-to-cart-dialog .btn-primary');
+    cy.window().then(win => {
+      expect((<any>win).Y_TRACKING.eventLayer.length).to.equal(2);
+      expect((<any>win).Y_TRACKING.eventLayer[1]['name']).to.equal(
+        'CartSnapshot'
+      );
+      const cartPayload = JSON.stringify(
+        (<any>win).Y_TRACKING.eventLayer[1]['data']
+      );
+      expect(cartPayload).to.include('cart');
+      expect(cartPayload).to.include('code');
+    });
+  });
+  it('should send an additional CartChanged event on modifying the cart', () => {
+    goToProductPage();
+    cy.get('cx-add-to-cart button.btn-primary').click();
+    cy.get('cx-added-to-cart-dialog .btn-primary').click();
+    cy.get('cx-cart-item-list')
+      .get('.cx-counter-action')
+      .contains('+')
+      .click();
+    cy.wait(500);
```

Comment:
what's the reason for waiting .5s?

If it's for cart to be updated, then you should wait for the `GET` request for the `cart` because we don't know if the server will be flimsy and not process it within that timeframe.

*I mentioned above, because I got it to fail and the expect fail bc 3 was not 4 (it didn't add the product)